src/CitizenProject/CitizenProjectUtils.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace AppBundle\CitizenProject;
+
+use AppBundle\Utils\BaseGroupUtils;
+
+class CitizenProjectUtils extends BaseGroupUtils
```

Comment:
What's the point of having empty children classes? IMHO you should only keep the base one.

Reply:
Done.